pkg/controller/metadata/metadata_controller.go

```diff
@@ -1,12 +1,9 @@
 /*
 Copyright 2019 GDS.
-
```

Comment:
Did something do this automatically? This is a standard license so I don't think we should be faffing with its contents, if it is only whitespace :-)

Reply:
Could have been automatic, could have been me! either way, I'll stick it back in (and hope it stays stuck.)